polyinterface/polyinterface.py

```diff
@@ -25,6 +25,12 @@
 from threading import Thread
 import warnings
 
+PY2 = sys.version_info[0] == 2
+
+if PY2:
+    string_types = basestring,
```

Comment:
do you need comma at the end of the line here?

Reply:
I had noticed that on the stack overflow example, and meant to pull them off both lines.